apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.html

```diff
@@ -21,3 +21,8 @@
     [assetIdentifier]="addToBundleIdentifier"
     (cancel)="addToBundleIdentifier = null"
 ></dot-add-to-bundle>
+
+<dot-add-to-menu
```

Comment:
Shouldn't we throw an `ngIf` here to remove from the DOM this on close? That way, the internal form gets destroyed.